src/contributing/adding_new_operation_how_to.md

```diff
@@ -0,0 +1,108 @@
+# Adding operations to Parsec
+
+This mini how-to will cover how to add operations to the backend and frontend of Parsec. The steps
+for adding operations to a backend will differ between providers, this guide will cover Mbed Crypto.
+The steps for adding operations to the frontend should be the same for all backend providers.
+
+## Backend (Mbed Crypto)
```

Comment:
For the section names, here is what I can propose:

```
## Operation specification
### parsec-book
### parsec-operations

## Parsec Rust Interface
With `parsec-interface-rs`

## Parsec Rust Client

## Parsec Service
### psa-crypto-sys
### psa-crypto
### parsec
``` 

What do you think?

Reply:
Could work, but we'd want to have a note saying what order it should be done in, right? The current order is the way I'd do it, now that I know how it all fits together and works.

Reply:
The order is the same to what you did, just the `psa-crypto*` crates grouped with `parsec` when doing the `Parsec Service` as I thought it would make sense to group those things together.

My main point was to give an idea about how section names would look like if you remove the `Frontend`/`Backend` names :)